automated/android/noninteractive-tradefed/setup.sh

```diff
@@ -22,6 +30,7 @@ case "${dist}" in
         ;;
 esac
 
+chech_python_version "$(python --verison)" "3.8" "Error"
```

Comment:
FYI, if like running Android11 cts/vts, python3.8 is not required, in fact it's just required for the AOSP master from now on. In the case of running Android11 cts/vts on Ubuntu18.04, python3.8 needs to be installed manually, otherwise it will fail here.